README.md

```diff
@@ -110,86 +110,122 @@ bash test/test_cleanup.sh
 [miRBase][mirbase] according to your instructions. Resources are then processed
 to prepare indexes and other contingent resources that will be used in later
 steps.
 
-The scheme below is a visual representation of an example run of the
-**_PREPARE_** workflow:
-
-> ![rule-graph-prepare][rule-graph-prepare]
-
 #### _MAP_
 
-The second workflow, **_MAP_** aligns the user-provided short read smallRNA-seq
-libraries against the references generated with the **_PREPARE_** workflow. For
+The second subworkflow, **_MAP_**, aligns the user-provided short read smallRNA-seq
+libraries against the references generated with the **_PREPARE_** subworkflow. For
 increased fidelity it uses two separate aligning tools, [Segemehl][segemehl]
 and our in-house tool [Oligomap][oligomap]. In both cases, reads are aligned
 separately to the genome and the transcriptome. Afterwards, alignments are
 merged in a way that only the best alignment (or alignments) of each read are
 kept.
 
-The scheme below is a visual representation of an example run of the **_MAP_**
-workflow:
-
-> ![rule-graph-map][rule-graph-map]
-
 #### _QUANTIFY_
 
-The third and final workflow, **_QUANTIFY_** quantifies miRNA expression by
-intersecting the alignments from the **_MAP_** workflow with the annotations
-generated in the **_PREPARE_** workflow. Intersections are computed with
+The third and final subworkflow, **_QUANTIFY_**, quantifies miRNA expression by
+intersecting the alignments from the **_MAP_** subworkflow with the annotations
+generated in the **_PREPARE_** subworkflow. Intersections are computed with
 [`bedtools`][bedtools] for one or multiple of mature, primary transcripts and
 isomiRs. Reads consistent with each miRNA are counted and tabulated.
 
-The scheme below is a visual representation of an example run of the
-**_QUANTIFY_** workflow:
 
-> ![rule-graph-quantify][rule-graph-quantify]
+The schema below is a visual representation of
+an example run of the whole workflow:
+
+> ![rule-graph][rule-graph]
 
-### Running the workflows
 
-Assuming that you are currently inside the repository's root directory, change
-to the run root directory:
+### Running the workflow
+
+Assuming that you are currently inside the repository's root directory, 
+create a directory from which you will run your workflow and
+name it whatever you want e.g., `MY_ANALYSIS` and 
+head to it.
 
 ```bash
-cd RUNS
+mkdir MY_ANALYSIS
+cd MY_ANALYSIS
 ```
+Now within the `MY_ANALYSIS/` directory, 
+create a new directory to store you library; 
+name it as you like, e.g. `my_lib` and move there.    
 
-Now make a clean copy of the `JOB` directory and name it whatever you want,
-e.g., `MY_ANALYSIS`:
+```bash
+mkdir my_lib
+cd my_lib
+```
+Place your library file(s) here. 
+In addition, create a sample table. 
+Fill it with the correct entries. 
+You can look at the `test/test_files/sample_table.csv` 
+to get an idea of what this file must look like.
```

Comment:
Users shouldn't only "get an idea" but be certain they understand the requirements of the input files. So apart from linking to the example, I think you should also describe the expected format (tab-separated, which columns are required...)